ec2/main.tf

```diff
@@ -0,0 +1,136 @@
+#Create Security Groups for public instances
+resource "aws_security_group" "public_ec2_sg" {
+  name = "public_ec2_sg"
+  description = "Allows TLS inbound traffic and outbound traffic"
+  vpc_id = aws_vpc.practice_vpc.id
+
+/*
```

Comment:
Also please clean up all comments within the resource block

Reply:
Yes, I have done that. I moved all comments to before the resource block.

Reply:
Oh ok, the changes have not been committed i guess

Reply:
Yea... I will push my items now... Had to stand up from my laptop for a bit.